### ISSUE TITLE
Embedded iframe does not display 12 hour time

### ISSUE THREAD

**Issue by zanton173** [niccokunzmann/open-web-calendar#220]
Not the same as issue 207 as I am working with an iframe using https://open-web-calendar.hosted.quelltext.eu/index.html?language=en online tool. ending_hour input does not change the timing of my calendar to 12 hour format from 24 hour format.

**Comment by niccokunzmann** [comment]
Hi, thanks for reporting! It seems that #207 aims at changing the hour format. Changing the ending hour is used to not display the end of the day in case e.g. all events are around morning.

Could you provide a screen shot so we know better that we talk about the same?

**Comment by zanton173** [comment]
Thanks for the quick response! Here are the current and desired screenshots. 
![current](https://github.com/niccokunzmann/open-web-calendar/assets/6892605/472fb02e-811a-4e89-9c24-4476bbb8dee5)
![desired](https://github.com/niccokunzmann/open-web-calendar/assets/6892605/c11e7ffa-7c8f-45f8-948e-ff1e5ea26a55)

**Comment by niccokunzmann** [comment]
It looks like exposing a way to change the time format might solve this. I am wondering if changing 19:00 to 7:00 would be common. So adding common choices and a free field in the configuration page might do it.

**Comment by zanton173** [comment]
Are you hosting https://open-web-calendar.hosted.quelltext.eu/index.html?language=en? Do you plan on creating this configuration field?

**Comment by niccokunzmann** [comment]
Yes, I am hosting this. This is one version you can use and people can 
host it themselves. It is open-source so people can contribute and it is 
hosted by me so people can use it without having hurdles like money or 
sign-up. If I have a plan. I will see! I am already thinking about how 
to implement this :)

**Comment by janiececald** [comment]
Hello @niccokunzmann, have you implemented this change to the configuration page?

**Comment by niccokunzmann** [comment]
Hi, have a look at https://github.com/niccokunzmann/open-web-calendar/pull/266. I think, this might help. You can then change the `hour_format` parameter to set the time format that you would like to see.

**Comment by niccokunzmann** [comment]
The possibility to use the 12h format is live at https://open-web-calendar.hosted.quelltext.eu/.

What I wonder is: Should the hour format be translated? People in Germany are used to another hour format than those in the UK for example.

**Comment by niccokunzmann** [comment]
I am closing this as you can now choose the hour format. If you would like to discuss this further, please comment!
